plugins/http_plugin/tests/unit_tests.cpp

```diff
@@ -604,6 +604,7 @@ BOOST_FIXTURE_TEST_CASE(bytes_in_flight, http_plugin_test_fixture) {
          boost::beast::http::request<boost::beast::http::empty_body> req(boost::beast::http::verb::get, "/4megabyte", 11);
          req.keep_alive(true);
          req.set(http::field::host, "127.0.0.1:8891");
+         s.wait(boost::asio::ip::tcp::socket::wait_write);
```

Comment:
What is the purpose of this and the `wait_read` below? Why would the socket not be writable here? And if it isn't writable why does it matter before a sync write is issued?

Reply:
Not needed. I'll remove.